chapter01/1.1 - Is Unique/isUnique.js

```diff
@@ -35,8 +35,20 @@ function everyCharUnique(str) {
   return true;
 }
 
+function everyCharUniqueV2(str){
+  //Only needs to run through the array once
+  for (let i = 0, len = str.length; i < len; i++){
+    if (i !== str.lastIndexOf(str.charAt(i))){
```

Comment:
Actually `.lastIndexOf` is loop, so still nested loop...

Reply:
Then it is a cleaner, nested solution :)

Reply:
Could you try and do a re-write without using inbuilt functions like `lastIndexOf` and `charAt` since both of them must have some cost.